yarn.lock

```diff
@@ -30,590 +30,667 @@
+    "@lumino/messaging" "^1.4.3"
+    "@lumino/signaling" "^1.4.3"
+    "@lumino/widgets" "^1.16.1"
     lodash.escape "^4.0.1"
-    marked "^0.8.0"
-
-"@jupyterlab/services@^5.1.0":
-  version "5.1.0"
-  resolved "https://registry.yarnpkg.com/@jupyterlab/services/-/services-5.1.0.tgz#891607d87cbe9413219cfe4ceb9905193b85b657"
-  integrity sha512-xhtDvAdgw+sWNSbpkExCYyJbHxlwhiZYqc07+zhOdYrpxO19k/ZmmyNoYCyfvNLcMQ4JWVBoczI714u1QNLj4w==
-  dependencies:
-    "@jupyterlab/coreutils" "^4.1.0"
-    "@jupyterlab/nbformat" "^2.1.0"
-    "@jupyterlab/observables" "^3.1.0"
-    "@jupyterlab/settingregistry" "^2.1.0"
-    "@jupyterlab/statedb" "^2.1.0"
-    "@lumino/algorithm" "^1.2.3"
-    "@lumino/coreutils" "^1.4.2"
-    "@lumino/disposable" "^1.3.5"
-    "@lumino/polling" "^1.1.1"
-    "@lumino/signaling" "^1.3.5"
+    marked "^2.0.0"
+
+"@jupyterlab/services@^6.0.9":
+  version "6.0.9"
+  resolved "https://registry.yarnpkg.com/@jupyterlab/services/-/services-6.0.9.tgz#70a10d7f6883b8fafff81216663d96858b0cf46b"
+  integrity sha512-zeN9roqwbYo6b2I5BXWx+Mr4KzTpe2UcVwrcAGw9NXqIieb0ZnvtHqtNj/vcHCM2xQKuPup9W1X1bE5b3wF5Yw==
+  dependencies:
+    "@jupyterlab/coreutils" "^5.0.6"
+    "@jupyterlab/nbformat" "^3.0.6"
+    "@jupyterlab/observables" "^4.0.6"
+    "@jupyterlab/settingregistry" "^3.0.6"
+    "@jupyterlab/statedb" "^3.0.6"
+    "@lumino/algorithm" "^1.3.3"
+    "@lumino/coreutils" "^1.5.3"
+    "@lumino/disposable" "^1.4.3"
+    "@lumino/polling" "^1.3.3"
+    "@lumino/signaling" "^1.4.3"
     node-fetch "^2.6.0"
     ws "^7.2.0"
 
-"@jupyterlab/services@https://github.com/platiagro/jupyterlab/releases/download/v0.2.0platiagro-v2.1.2/jupyterlab-services-5.1.0.tgz":
-  version "5.1.0"
-  resolved "https://github.com/platiagro/jupyterlab/releases/download/v0.2.0platiagro-v2.1.2/jupyterlab-services-5.1.0.tgz#e2e71a0d57cdf8a56512ffd3d2acf21e4e32ee39"
-  dependencies:
-    "@jupyterlab/coreutils" "^4.1.0"
-    "@jupyterlab/nbformat" "^2.1.0"
-    "@jupyterlab/observables" "^3.1.0"
-    "@jupyterlab/settingregistry" "^2.1.0"
-    "@jupyterlab/statedb" "^2.1.0"
-    "@lumino/algorithm" "^1.2.3"
-    "@lumino/coreutils" "^1.4.2"
-    "@lumino/disposable" "^1.3.5"
-    "@lumino/polling" "^1.1.1"
-    "@lumino/signaling" "^1.3.5"
+"@jupyterlab/services@https://github.com/platiagro/jupyterlab/raw/v0.2.0-jupyterlab-v3.0.16/packages/services/jupyterlab-services-v6.0.9.tgz":
+  version "6.0.9"
+  resolved "https://github.com/platiagro/jupyterlab/raw/v0.2.0-jupyterlab-v3.0.16/packages/services/jupyterlab-services-v6.0.9.tgz#a5171dff294065f9a0d406b3fa4da6b9b81b6647"
+  dependencies:
+    "@jupyterlab/coreutils" "^5.0.6"
+    "@jupyterlab/nbformat" "^3.0.6"
+    "@jupyterlab/observables" "^4.0.6"
+    "@jupyterlab/settingregistry" "^3.0.6"
+    "@jupyterlab/statedb" "^3.0.6"
+    "@lumino/algorithm" "^1.3.3"
+    "@lumino/coreutils" "^1.5.3"
+    "@lumino/disposable" "^1.4.3"
+    "@lumino/polling" "^1.3.3"
+    "@lumino/signaling" "^1.4.3"
```

Comment:
Instala o @jupyterlab/services original e o alterado simultaneamente? :O

Reply:
Vou verificar, acho que se excluir o yarn.lock e criar de novo isso é corrigido

Reply:
Mesmo recriando o yarn.lock ainda aparece os dois, provavelmente porque outras dependências da extensão possuem o `@jupyterlab/services` como dependência usam a versão no package.json, ao invés de uma URL do GitHub, como estamos fazendo.

![image](https://user-images.githubusercontent.com/36794274/127183951-c4dd1f11-df4b-497e-ba72-4a395f37f445.png)

Rodando `yarn why @jupyterlab/services` para saber quem depende dessa biblioteca, olha:

![image](https://user-images.githubusercontent.com/36794274/127185008-d08bb628-a5a1-449f-b1c6-de1eda9b71ab.png)

Como declaramos usando a URL nas dependências ele deve usar ela preferencialmente em cima das outras, mas esse problema do yarn duplicar as dependências já é conhecido. Descobri que antes o yarn não resolvia esse problema, então surgiu ferramentas como o [yarn-deduplicate](https://yarnpkg.com/package/yarn-deduplicate), mas o yarn v2 tem um comando nativo pra resolver isso: [yarn dedupe](https://yarnpkg.com/cli/dedupe).

Vou tentar usar esse comando para testar.

Reply:
Não é necessário rodar o comando, pois:

yarn dedupe v1.22.10
error The dedupe command isn't necessary. `yarn install` will already dedupe.